float32 x
float32 y
float32 width
float32 height
float32 angle
float32 distance
bool reset
bool tracking